docs/src/components/code-block.astro

```diff
@@ -0,0 +1,21 @@
+---
+import { Code } from "astro:components";
+
+import { CopyButton } from "@/components/copy-button";
+
+interface Props {
+  code: string;
+  lang?: "bash" | "js";
+  copy?: boolean;
+}
+
+const { code, lang = "js", copy = false } = Astro.props;
```

Comment:
We should use "ts" as the default language, we sometimes use typescript code in our examples.
```suggestion
interface Props {
  code: string;
  lang?: "bash" | "ts";
  copy?: boolean;
}

const { code, lang = "ts", copy = false } = Astro.props;
```